Backend/dev/migrations/special-zone/special-zone.sql

```diff
@@ -0,0 +1,19 @@
+CREATE FUNCTION atlas_special_zone.no_overlaps_in_special_zone(id character(36), g geometry(MultiPolygon,4326))
+RETURNS boolean AS $$
+SELECT NOT EXISTS (
+  SELECT 1 FROM atlas_special_zone.special_zone a
+  WHERE a.id != $1
+    AND a.geom && g
+    AND ST_Relate(a.geom, g, '2********'));
+$$ LANGUAGE sql;
+
+ALTER FUNCTION atlas_special_zone.no_overlaps_in_special_zone(character(36), geometry) OWNER TO atlas_special_zone_user;
+
+ALTER TABLE atlas_special_zone.special_zone ADD CONSTRAINT no_overlaps CHECK (atlas_special_zone.no_overlaps_in_special_zone(id, geom));
+
+CREATE FUNCTION atlas_special_zone.geojson_to_bin(geojson text)
```

Comment:
Could you look into implementing this in code itself?
Found some references on how to achieve this - https://gis.stackexchange.com/questions/423683/convert-geojson-geometry-to-new-coordinate-system-and-wkb-format-in-python

Can be done in next iteration